README.md

```diff
@@ -1,112 +1,204 @@
+  Additionally, semver-minor changes will be delayed and grouped as much as is practical in LTS release lines in order to ease the upgrade path for users of that line.
+  Any semver-minor commits proposed for inclusion in an LTS release line will require:
+  - sign-off from at least one active member of the LTS Working Group;
+  - no objection from any active member of the LTS Working Group
+* Receive semver-patch commits, i.e. those not labelled either semver-major or semver-minor as appropriate.
+  Any semver-patch commits proposed for inclusion in an LTS release line will require:
+  - sign-off from at least one active member of the LTS Working Group;
+  - no objection from any active member of the LTS Working Group
+
+### Acceptable upgrades to the V8 dependency
+
+The Node.js JavaScript runtime, [V8](https://developers.google.com/v8/), is part of the Node.js C++ API. Therefore, changes to the C++ API of V8 have a flow-on effect to the Node.js package ecosystem and must be treated in a similar way to changes to the core of Node.js itself. In general, major versions of V8 (e.g. 5.1, 5.2, 5.3, etc. — note that these are not semver) are considered _breaking changes_ for Node.js users due to the regular changes in the C++ [API](https://en.wikipedia.org/wiki/Application_programming_interface) (application programming interface) and [ABI](https://en.wikipedia.org/wiki/Application_binary_interface) (application binary interface) even if the JavaScript API remains stable.
+
+All new semver-major increments start out with a fresh API and ABI from their `x.0.0` release. While the C++ API may change over time according to semver rules (e.g. introducing new features that are backward-compatible—"semver-minor"), the **ABI must strictly remain stable over the life of that major version**, even as it moves through the various stages ("Current", "Active LTS" and "Maintenance").
+
+Release lines in "Active LTS" and "Maintenance" will be pinned to a single major version of V8 during their lifetime in order to maximise stability. Bug fixes and improvements may be introduced, either by downstreaming, backporting, or by original authorship, according to the same policy for Node.js changes for LTS release lines.
+
+Release lines in "Current" _may_ receive updates to their V8 dependency. This will occur ***only if*** it can be done in a way that is not semver-major and does not break the stable ABI of the current major version. This may be achieved by either receiving an ABI-stable update from upstream or by applying floating patches to the dependency to ensure ABI-stability. Implementation details are left up to the Node.js V8 team.
+
+Upgrades to V8 in a "Current" release line should not occur within two months of that release line moving in to "Active LTS". i.e. upgrades to V8 should not occur from August through to the transition of "Current" into LTS in October.
+
+### Changes acceptable for "Active LTS"
```

Comment:
Yeah, but I couldn't come up with any! So I left it above as:
  - sign-off from at least one active member of the LTS Working Group;
  - no objection from any active member of the LTS Working Group

Meaning that our reasoning can evolve over time and perhaps we can codify it when we end up with an established pattern; I just don't see that now in how we're working and don't want to prematurely make things strict. See https://github.com/nodejs/node/issues/7714 (backport `fs.mkdtemp()`) for an example of this, I'm not comfortable ruling it in or out because I don't see a clear stance we're prepared to make as a group.

Unless you want to propose some wording that gets us in the right direction?